src/Carbon.ts

```diff
@@ -60,15 +63,18 @@ export class Class extends AbstractContext.Class {
 
 	protected _baseURI:string;
 
-	constructor( domain:string, ssl?:boolean, settings?:Settings.Class );
+	_messaging:Messaging.Service.Class;
+
 	constructor( domain:string, ssl:boolean = true, settings?:Settings.Class ) {
 		super();
-		domain = RDF.URI.Util.hasProtocol( domain ) ? RDF.URI.Util.removeProtocol( domain ) : domain;
-		domain = Utils.S.endsWith( domain,  "/" ) ? domain : domain + "/";
+		domain = RDF.URI.Util.hasProtocol( domain ) ? domain.substring( domain.indexOf( "://" ) + 3 ) : domain;
```

Comment:
Please prefer the use of methods rather than "clever" solutions that are more difficult to read. In this case change the method implementation if you think it should work differently.

Reply:
I'll change the method implementation to also remove the `://` part.